src/services/axiosSetup.ts

```diff
@@ -22,11 +22,13 @@ axios.interceptors.response.use(
   async (error) => {
     const originalRequest = error.config;
 
-    if (error.response.status === 401 && !originalRequest._retry) {
-      originalRequest._retry = true;
+    if (!originalRequest._retryCount) {
```

Comment:
What makes us sure we won't go into infinite loop?

Reply:
done